actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -39,6 +39,19 @@ def capabilities
           end
       end
 
+      def preload
+        # driver_path can be configured as a proc. The webdrivers gem uses this
+        # proc to update web drivers. Running this proc early allows us to only
+        # update the webdriver once and avoid race conditions when using
+        # parallel tests.
```

Comment:
This class is private so I think either this comment should be moved above preload and it won't load in the docs anyway, or should be removed.